keyboards/splitish/rules.mk

```
@@ -0,0 +1,15 @@
+# MCU name
+MCU = atmega32u4
+
+BOOTLOADER = caterina
+
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
```

```suggestion
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
```
We've moved away from including firmware filesize changes in board-level `rules.mk` as it's misleading, because they are highly dependent on architecture and constantly changing.